lambdas/revoke-vcs/src/main/java/uk/gov/di/ipv/core/revokevcs/RevokeVcsHandler.java

```diff
@@ -86,6 +86,7 @@ public void handleRequest(InputStream inputStream, OutputStream outputStream, Co
                 new ObjectMapper()
                         .readValue(inputStream, VcsActionRequest.class)
                         .getUserIdCriIdPairs();
+        result = new RevokeVcsResult();
```

Comment:
I'd still prefer this to be a new variable in the handler method, but this _should_ be fine...

(It'll only be bad if anything above it throws...)